tests/Database/DatabaseEloquentBuilderTest.php

```diff
@@ -109,30 +109,6 @@ public function testFirstMethod()
         $this->assertEquals('bar', $result);
     }
 
-    public function testGetMethodLoadsModelsAndHydratesEagerRelations()
```

Comment:
Basically, applyScopes clones the instance so the mocking expectations break. From my original post:

"I had to delete 2 tests due to heavy mocking which can't be done on a cloned instance. And integration tests already cover that scenario, so they're not really needed."